### PR TITLE
Update getTimestamp function and tests

### PR DESCRIPTION
**Pull request by cdeborja**
- Fixes `getTimestamp` function to accommodate for `null` and garbage input strings. 
- Adds specs for testing `getTimestamp` function

### CODE REVIEW

File: test/lib/query.spec.js [hunk 1]
```diff
@@ -293,4 +294,80 @@ describe('Pimp My Sql :: Query', function() {
...
+    it('should return null for null input', () => {
+
+      const nullInput = null
+      const result = Query.getTimestamp(nullInput)
+
+      expect(result).to.eql(null);
+
+    })
+
+    it('should return null for garbage input', () => {
```
Comment (clintonbess) [inline test/lib/query.spec.js]: 🗑

File: test/lib/query.spec.js [hunk 2]
```diff
@@ -293,4 +294,80 @@ describe('Pimp My Sql :: Query', function() {
...
+
+      const input = 'thisisgarbateinput'
```
Comment (clintonbess) [inline test/lib/query.spec.js]: garbate?

**Review by clintonbess — APPROVED** [review]

**Comment by clintonbess** [comment]
What about the case where the input is empty?

File: test/lib/query.spec.js [hunk 3]
```diff
@@ -293,4 +294,89 @@ describe('Pimp My Sql :: Query', function() {
     })
   })
 
+  describe('::getTimestamp', () => {
+    it('should return a proper date and time for a unix timestamp', () => {
+
+      const input = 1515188138
```
Comment (scull7) [inline test/lib/query.spec.js]: What happens if a millisecond based timestamp is given?

**Comment by jxbadam** [comment]
You may haz the build failure ...

https://travis-ci.org/influentialpublishers/pimp-my-sql/jobs/325657012

We <3 timezones!

File: lib/query.js [hunk 4]
```diff
@@ -21,13 +21,23 @@ const idMatch = _.curry((id, obj) => id + '' === obj.id + '');
 
 const dateIsUnix = _.compose(_.equals(0), _.modulo(_.__, 1));
 
+const dateIsMs = _.compose(
```
Comment (AsaAyers) [inline lib/query.js]: All this Ramda is unnecessary
```js
const dateIsMs = (num) => (parseInt(num) > LAGEST_UNIX_TIMESTAMP)
```

**Review by AsaAyers — CHANGES_REQUESTED** [review]

File: lib/query.js (hunk 4 above)
Comment (cdeborja) [reply lib/query.js]: but... the ramda!

Will update, thanks!

**Review by jxbadam — APPROVED** [review]
:shipit: :fire:

Before the coveralls strike again!

File: lib/query.js [hunk 5]
```diff
@@ -22,12 +22,15 @@ const idMatch = _.curry((id, obj) => id + '' === obj.id + '');
 const dateIsUnix = _.compose(_.equals(0), _.modulo(_.__, 1));
 
 
-const momentFromString = (date) => moment(new Date(date));
+// dateInMs :: Int -> Bool
+const dateIsMs = (num) => (parseInt(num) > LARGEST_UNIX_TIMESTAMP)
```
Comment (mszymulanski) [inline lib/query.js]: Don't ever compare different units without converting them to something common. In the case of a problem that you are trying to solve here, there is no reliable solution.

The problem of passing in an integer which may be milliseconds or seconds probably will be okay with our current system, but it is inherently wrong, buggy and evil. You can put a big, big scary comment on top of this function and a reference comment in every place you use this function. In the comment, include boundaries of when this function works. 

But again, don't ever compare different units. 

Perfectly, we should only ever use seconds as a timestamp.

Comment (mszymulanski) [reply lib/query.js]: Also, pimp-my-sql is a public library. If somebody sees this function, they will mock us for ever and ever. I wouldn't put my username in git on that code.

File: test/lib/query.spec.js (hunk 3 above)
Comment (mszymulanski) [reply test/lib/query.spec.js]: They you smack that developer for using milliseconds as a timestamp. The industry standard is seconds.

**Pull request merged**